hz: 10 #周期
laser_step: 3 #何本ごとにレーザーの値をみるか
ignore_dist: 0.01 #外れ値
